core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1367,6 +1367,18 @@ public static String getSyncMetricName(long mountId) {
           .setDescription("The number of active write-RPCs managed by workers")
           .setMetricType(MetricType.COUNTER)
           .build();
+  public static final MetricKey CLUSTER_BLOCK_READER_THREAD_ACTIVE_COUNT =
+      new Builder("Cluster.BlockReaderThreadActiveCount")
+          .setDescription("The total number of block read threads "
+              + "that are actively executing tasks in reader thread pool by workers")
+          .setMetricType(MetricType.COUNTER)
+          .build();
+  public static final MetricKey CLUSTER_BLOCK_WRITER_THREAD_ACTIVE_COUNT =
```

Comment:
```suggestion
  public static final MetricKey CLUSTER_BLOCK_WRITER_ACTIVE_THREADS_COUNT =
```

Reply:
I use `CLUSTER_BLOCK_WRITER_THREAD_ACTIVE_COUNT` here because it's consistent with the existing metric `WORKER_BLOCK_READER_THREAD_ACTIVE_COUNT`. I feel it's better to change both if really needed.